src/components/FlashMessage/FlashMessage.test.js

```diff
@@ -36,7 +37,7 @@ it('should render a closable message', () => {
 it('should render a message which closes itself after the given time', async () => {
   const message = 'this is the message'
   const time = 10
-  const { queryByText, getByText } = render(
+  const { queryByText, getByText } = await renderAndWait(
```

Comment:
what was the reason for this change?

Reply:
When updating the `createOfferRequest` tests, which trigger a flash message, I received an error with this flash message test and used the `await renderAndWait` to ensure the flash message wasn't prompted until all the mock queries were complete.  Since then, I updated the `createOfferRequest` tests with a missing mock Query, which also resolved .  Consequently, this test will work with or without the `renderAndWait` now.

Reply:
great, will you put it back to regular `render`?

Reply:
if this works with `render` we should use that.